iterators_task_2/student/include/impl.h

```diff
@@ -5,11 +5,42 @@ template <class Container>
 struct accumulator
 {
     /** @todo Iterator traits*/
+    using difference_type = typename Container::difference_type;
```

Comment:
for output iterators usually all traits are set to void (except iterator category)